app/views/hyrax/base/_form_visibility_component.html.erb

```diff
@@ -1,6 +1,6 @@
-<% if embargo_enforced?(f.object) %>
+<% if embargo_enforced?(f.object) && !f.object.new_record? %>
```

Comment:
i'm wondering whether the `embargo_enforced?` helper shouldn't take the `new_record?` case into account? then this `&&` could be inlined there and kept out of the view?

the other place this is used is `_form_permission.html.erb` and, at first glance, i'm guessing that partial will have similar issues in the same cases as this one. it might be fair to say that `embargo/lease_enforced?` isn't true when the object isn't yet saved.

Reply:
This makes sense, and a new commit is incoming!